packages/pages/src/App.js

```diff
@@ -70,6 +86,16 @@ export default provideState({
       this.state.content = "";
       await this.effects.refreshEntries();
     },
+    async deleteEntry() {
+      await fetch("/api/", {
+        method: "post",
+        body: format.request(0, "deleteEntry", {
+          id: this.state.id,
+        }),
+      });
```

Comment:
You need to parse and check the response to be sure the entry has been correctly deleted.